xbmc/cores/VideoPlayer/DVDSubtitles/DVDSubtitlesLibass.cpp

```diff
@@ -92,11 +92,93 @@ void CDVDSubtitlesLibass::Configure()
   ass_set_margins(m_renderer, 0, 0, 0, 0);
   ass_set_use_margins(m_renderer, 0);
 
-  // Libass uses system font provider (like fontconfig) by default in some
-  // platforms (e.g. linux/windows), on some other systems like android the
-  // font provider is currently not supported, then an user can add his
-  // additional fonts only by using the user fonts folder.
-  ass_set_fonts_dir(m_library, CSpecialProtocol::TranslatePath(FONT::FONTPATH::USER).c_str());
+  std::string fontsDir = CSpecialProtocol::TranslatePath(FONT::FONTPATH::USER);
+  int fontProvider = ASS_FONTPROVIDER_AUTODETECT;
+  std::string fontconfigPath; // Keep string alive for lifetime of fontconfigConfig pointer
+  const char* fontconfigConfig = nullptr;
+
+#if defined(TARGET_ANDROID)
+  // Use fontconfig with custom config pointing to Android system fonts.
+  // Fontconfig provides automatic font fallback for all scripts (Malayalam, Tamil, Chinese, etc.).
```

Comment:
> // Use fontconfig with custom config pointing to Android system fonts.

All following code is clearly a _workaround / hack_ since Libass does not currently support native Android "fontconfig", this must be clearly specified, so instead to remove in full the original comment, it must be readapted

> // Fontconfig provides automatic font fallback for all scripts (Malayalam, Tamil, Chinese, etc.).

this comment is not required, is an intrinsic feature